### PR TITLE
Namespace update from HDC-FlaskDeID to FlaskPHI_ID

### PR DESCRIPTION
**Pull request by WhiteAu**
HDC-FlaskDeID -> FlaskPHI_ID
flaskdeid -> flaskphiid
medlp -> compmed
  -rip out /annotate/ affix in the medlp endpoints
deid -> identifyphi
  -rip out /annotate/ affix in the deid endpoints

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
 
-
+## General Installation
+venv venv
+source venv/bin/activate.sh
+python setup.py install
+python flaskphiid/run.py -e 0.0.0.0 -p 5000 #run flask app at endpoint 0.0.0.0 on port 5000
 
 ## test strings
```
Comment (esilgard) [inline README.md:14]: can I request a PHI worthy test string in the README?

note_text = 'Mr. Doe was diagnosed with Stage III adenocarcinoma of the left lung on 3/14/2018'
summin like that

File: flaskphiid/__init__.py [hunk 2]
```diff
@@ -17,8 +17,8 @@
 # load medlp interface
 from amazonserviceinterface.MedLPServiceInterface import MedLPServiceInterface
 import clinicalnotesprocessor.JSONParser as JSONParser
```
Comment (esilgard) [inline flaskphiid/__init__.py]: from compmed.CompMedServiceInterface import CompMedServiceInterface
import utils.json_parser_utils as JSONParser

File: flaskphiid/__init__.py [hunk 3]
```diff
@@ -17,8 +17,8 @@
...
-medlpInterface = MedLPServiceInterface(JSONParser.xform_dict_to_json)
-#load HutchNER interaface
+compmedInterface = MedLPServiceInterface(JSONParser.xform_dict_to_json)
```
Comment (esilgard) [inline flaskphiid/__init__.py]: compmedInterface = CompMedServiceInterface(...

File: README.md (hunk 1 above)
Comment (WhiteAu) [reply README.md:14]: should already be one in there under the $ test strings heading

File: setup.py [hunk 4]
```diff
@@ -37,29 +37,26 @@ def get_env_variable(var_name, default=False):
...
+    name='FlaskPHI_ID',
     version='0.1',
-    packages=['flaskdeid', 'test', 'test.flaskdeid',],
-    url='https://github.com/WhiteAu/FlaskML',
-    install_requires=['clinicalpreprocessing',
-                      'amazonserviceinterface',
+    packages=['flaskphiid',],
+    test_packages=['test', 'test.flaskphiid',],
+    url='https://github.com/FredHutch/FlaskPHI_ID',
+    install_requires=['amazonserviceinterface',
```
Comment (esilgard) [inline setup.py]: compmed

File: test/flaskphiid/test_compmed.py [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+import flaskphiid
 import unittest
 import json
 
 from flask import g, session, Response
-from flaskdeid import create_app
+from flaskphiid import create_app
 from unittest.mock import patch
-import flaskdeid.medlp as medlp
+import flaskphiid.compmed as medlp
```
Comment (esilgard) [inline test/flaskphiid/test_compmed.py]: import flaskphiid.compmed as compmed?

File: test/flaskphiid/test_compmed.py [hunk 6]
```diff
@@ -52,7 +52,7 @@ def test_annotate_empty_entity_text(self):
         self.assertEqual(result.status_code, 400)
 
 
-    @patch('flaskdeid.medlpInterface.get_entities')
+    @patch('flaskphiid.medlpInterface.get_entities')
```
Comment (esilgard) [inline test/flaskphiid/test_compmed.py]: compMedInterface

**Review by esilgard — APPROVED** [review]
coo coo
couple comments about comp med references

**Pull request merged**